src/types/member.ts

```diff
@@ -1,6 +1,7 @@
 export type Member = {
   member_id: string;
-  name?: string;
+  first_name?: string;
```

Comment:
Because this type is also used by `key.ts` we can't modify it's structure. We'll likely need a new type for now.

Reply:
It looks like it is being used incorrectly though. When I call the api I get a `first_name` and `last_name` in the response. Not a `name` property. 

<img width="624" alt="Screen Shot 2022-03-17 at 9 01 31 AM" src="https://user-images.githubusercontent.com/13695093/158832239-73e0de34-c820-4580-a23b-27a94acf3907.png">